src/Features/Core/Portable/MakeMemberStatic/AbstractMakeMemberStaticCodeFixProvider.cs

```diff
@@ -18,12 +18,12 @@ internal abstract class AbstractMakeMemberStaticCodeFixProvider : SyntaxEditorBa
     {
         internal sealed override CodeFixCategory CodeFixCategory => CodeFixCategory.Compile;
 
-        protected abstract bool IsValidMemberNode([NotNullWhen(true)] SyntaxNode? node);
+        protected abstract bool TryGetMemberDeclaration(SyntaxNode node, [NotNullWhen(true)] out SyntaxNode? memberDeclaration);
```

Comment:
There is no need for an abstract class here, only C# is implemented. But I didn't cleanup this.

Reply:
I think there's a lot of optimism that this could be in VB :) 

Fine to leave for now I think

Reply:
@ryzngard The situation is the exact opposite in VB.

For C#, a static class requires all members to be static
For VB, a module requires all members to be non-Shared (they're `Shared` implicitly).

There is a code fix for the VB situation:

https://github.com/dotnet/roslyn/blob/main/src/Features/VisualBasic/Portable/RemoveSharedFromModuleMembers/VisualBasicRemoveSharedFromModuleMembersCodeFixProvider.vb